packages/workspace/src/schematics/move/lib/update-jest-config.ts

```diff
@@ -50,14 +50,15 @@ export function updateJestConfig(schema: Schema): Rule {
           .read(rootJestConfigPath)
           .toString('utf-8');
 
+        const { libsDir, appsDir } = getWorkspaceLayout(tree);
         const findProject = new RegExp(
-          `'<rootDir>\/(libs|apps)\/${schema.projectName}'`,
+          `<rootDir>\/(${libsDir}|${appsDir})\/${schema.projectName}`,
           'g'
         );
 
         const newRootJestConfigContent = oldRootJestConfigContent.replace(
           findProject,
-          `<rootDir>/${destination}`
+          `'<rootDir>/${destination}'`
```

Comment:
You should remove the single quotes from here too. The final config would have double single quotes 😅